pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -60,8 +61,9 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	// create kubeadm init config
 	fns := []func() error{}
 
+	provider := fmt.Sprintf("%s", ctx.Provider)
```

Comment:
huh did I do that? This is a pretty roundabout way to call `ctx.Provider.String()` ...

Reply:
` type provider.Provider has not method or type String()` 
how does the stringer works with interfaces by the way? 
is not necessary to export it in the interface?

Reply:
ah, no so, the interface doesn't expose it, but `Sprintf` for `"%s"` will check if it can assert `fmt.Stringer` and call String anyhow.

this is why I did this 🙃